kata-containers-image/update.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+
+# Automation script to create specs to build kata-containers-image
+# Default image to build is the one specified in file versions.txt
+# located at the root of the repository.
+set -e
+
+source ../versions.txt
+source ../scripts/pkglib.sh
+
+SCRIPT_NAME=$0
+SCRIPT_DIR=$(dirname $0)
+PKG_NAME="kata-containers-image"
+VERSION=$kata_osbuilder_version
+
+GENERATED_FILES=(kata-containers-image.spec kata-containers-image.dsc debian.rules debian.control)
+STATIC_FILES=(LICENSE debian.compat debian.dirs kata-containers.tar.gz)
+
+# Parse arguments
+cli "$@"
+
+[ "$VERBOSE" == "true" ] && set -x
+PROJECT_REPO=${PROJECT_REPO:-home:${OBS_PROJECT}:${OBS_SUBPROJECT}/kata-containers-image}
+RELEASE=$(get_obs_pkg_release "${PROJECT_REPO}")
+((RELEASE++))
+[ -n "$APIURL" ] && APIURL="-A ${APIURL}"
+
+function check_image() {
+    [ ! -f "${SCRIPT_DIR}/kata-containers.tar.gz" ] && die "No kata-containers.tar.gz found!\nUse the build_image.sh script" || echo "Image: OK"
+}
+
+sed -i s/"kata_vm_image_version=${kata_vm_image_version}"/"kata_vm_image_version=${VERSION}"/ ${SCRIPT_DIR}/../versions.txt
```

Comment:
before approve this PR, why is needed this?

Reply:
I think this used to be `cc_vm_image_version` and was used to record the latest Clear Linux release our `clear-containers-image` package was based on.

However, now that osbuilder supports different distros and different agent configurations, I don't think we actually need to do this any more. Particularly, since we now create a summary file inside the image:

- https://github.com/kata-containers/osbuilder/pull/83

... the details of which are recorded when the user runs `kata-collect-data.sh`:

- https://github.com/kata-containers/runtime/pull/238